FRW/Modules/MultiRestore/ViewModel/RestoreMultiAccountViewModel.swift

```diff
@@ -52,9 +52,13 @@ class RestoreMultiAccountViewModel: ObservableObject {
                     MultiAccountStorage.shared.setBackupType(.multi, uid: selectedUserId)
                     HUD.dismissLoading()
                 } catch {
```

Comment:
Try with this, swift should able to catch specific error
```suggestion
               catch LLError.accountNotFound {
                  addKey(item: selectedUser)
                } catch {
                  log.error("switch account failed", context: error)
                  HUD.dismissLoading()
                  HUD.error(title: error.localizedDescription)
                  return
                }
```